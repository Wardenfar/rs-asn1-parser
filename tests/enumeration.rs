use asn1_parser::parse_asn1_file;

#[test]
fn enum_ok() {
    let m = parse_asn1_file(include_str!("../resources/enum-ok.asn1")).unwrap();
    insta::assert_json_snapshot!(m);
}

#[test]
fn enum_err_1() {
    let e = parse_asn1_file(include_str!("../resources/enum-err-1.asn1")).unwrap_err();
    insta::assert_display_snapshot!(e)
}
