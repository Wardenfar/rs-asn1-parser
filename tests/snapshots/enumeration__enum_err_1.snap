---
source: tests/enumeration.rs
assertion_line: 12
expression: e

---
Two items of the same enum conflict values
